superset/connectors/sqla/models.py

```diff
@@ -1253,10 +1254,19 @@ def get_sqla_query(  # pylint: disable=too-many-arguments,too-many-locals,too-ma
                     result.df, dimensions, groupby_exprs_sans_timestamp
                 )
                 qry = qry.where(top_groups)
+        if is_rowcount:
+            if not db_engine_spec.allows_subqueries:
+                raise QueryObjectValidationError("Database does not support subqueries")
```

Comment:
nit: can this surface to the UI? if use `_("Database ....")` if not ignore me :P